<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoCardDialog</class>
 <widget class="QDialog" name="VideoCardDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Video Card Configuration</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="driverBox">
     <property name="title">
      <string>Video Driver</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="autoDriverButton">
        <property name="text">
         <string>Use the default driver for the detected card.</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="manualDriverButton">
          <property name="text">
           <string>Select from the list:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="driverSelection">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string notr="true">radeon</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="depthBox">
     <property name="title">
      <string>Color Depth</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="autoDepthButton">
        <property name="text">
         <string>Determined by the driver</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="manualDepthButton">
          <property name="text">
           <string>Select from the list:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="depthSelection">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>16 Bit</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>24 Bit</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VideoCardDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VideoCardDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manualDriverButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>driverSelection</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manualDepthButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>depthSelection</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
